.github/workflows/build.yml

```diff
@@ -244,7 +244,7 @@ jobs:
         working-directory: lite
         run: |
           set -eux
-          mkdir -p content && cp ../examples/*.jcad ../examples/*.STEP ../examples/*.stl ../examples/Notebook.ipynb ./content
+          mkdir -p content && cp ../examples/*.jcad ../examples/*.STEP ../examples/*.stl ../examples/Notebook.ipynb ../examples/jcad.ipynb ./content
```

Comment:
```suggestion
          mkdir -p content && cp ../examples/*.jcad ../examples/*.STEP ../examples/*.stl ../examples/*.ipynb ./content
```

Reply:
i think i'll need this, since i'm not able to create a new file and using an existing file

Reply:
I'm not sure I get your comment here? Can you explain more what you mean?

Reply:
Oh sorry, i see your suggestion again and i explained myself wrong.
But still, since we don't want `occ.ipynb` on lite - I'm not sure we'd want to include `../examples/*.ipynb`.
Can i exclude just `occ.ipynb` instead?

Reply:
Ah right! Let's keep it that way then